scripts/delete.sh

```diff
@@ -51,7 +51,7 @@ fi
 # Ensure offline
 #--------------------------------------
 runner_status=$(curl -s -X GET ${base_api_url}/${runner_scope}/actions/runners?per_page=100  -H "accept: application/vnd.github.everest-preview+json" -H "authorization: token ${RUNNER_CFG_PAT}" \
-        | jq -M -j ".runners | .[] | [select(.name == \"${runner_name}\")] | .[0].status")
+        | jq -M -j ".runners | .[] | [select(.name == \"${runner_name}\")] | select(length > 0) | .[0].status")
```

Comment:
Hi @nhsieh, thank you very much for this PR!

Runner names are guaranteed to be unique within each repo. I don't know `jq` syntax super well and I don't see a reason to wrap the result of the select in an array like so: `[select(.name == \"${runner_name}\")]`.

What do you think of this instead?
```
| jq -M -j ".runners | .[] | select(.name == \"${runner_name}\") | .status")
```

Reply:
Yes @fhammerl, that works better. I've pushed an update. Thank you for the review!